### PR TITLE
Bugfix/zcs 1549

### PR DESCRIPTION
**Pull request by jasonyi-zimbra**
ZCS-1549 fixed local ip list tracking.

### CODE REVIEW

File: WebRoot/js/zimbraAdmin/servers/controller/ZaServerController.js [hunk 1]
```diff
@@ -265,26 +265,46 @@ function (params) {
...
+
+                    // bug ZCS-1549:  allow non-CIDR inet addresses
+                    var j = 0;
+                    var ipFound = false;
+
+                    while (validStr.search("/") < 0 && j < locals.length)
+                    {
+			if (validStr === locals[j].ipData.src)
+			{
+                                locals.splice(j,1);
```
Comment (SatyamKumarOnline) [inline WebRoot/js/zimbraAdmin/servers/controller/ZaServerController.js]: Indentation

**Comment by SatyamKumarOnline** [comment]
Please update testing notes

**Comment by jasonyi-zimbra** [comment]
@SatyamKumarOnline 

Tested cases created by IMT in Jira ZCS-1549 as well as CIDR and non-CIDR formats.

**Comment by jasonyi-zimbra** [comment]
Rebased and conflicts resolved.

**Comment by silentsakky** [comment]
Merged in https://github.com/Zimbra/zm-admin-console/commit/b0b3ce59618f5971dbcc5cc0ef1656f92e2e0b30

**Pull request closed without merging**